src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
-import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comment } from '../../types/Comment';
+
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList = ({ comments }: Props) => (
+  <div className="CommentList">
```

Comment:
the return word here can be removed `{comments.map(comment => <CommentInfo comment={comment} key={comment.id} />)}`